pkg/sidecar/config.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+	FederatedTrustDomain               string `hcl:"federated_trust_domain"`
+	FederationBundleFile               string `hcl:"federation_bundle_file"`
```

Comment:
Should there be an option to bundle it into the main ca file for those applications that can only be passed one?

Reply:
that makes sense, I just changed the code to do that instead of using a separated file

Reply:
Maybe a '*' for matching them all too. if there is a lot of them, it might be really painful to manually list them all.

Reply:
Do you know which go-spiffe package has a function to list the federated domains of a registration entry?

Reply:
I just changed the code a little. I set the federated_trust_domains parameter as a boolean. When it is set, all the bundles received in the svidResponse will be added to the CA bundle (in the current version of SPIFFE-helper, only the bundle associated to the svidResponse.ID TrustDomain is included).

There is no longer the need of specifying trust domains one by one.